tools/topology/sof-byt-codec.m4

```diff
@@ -47,39 +55,42 @@ PIPELINE_PCM_ADD(sof/pipe-low-latency-capture.m4,
 
 # playback DAI is SSP2 using 2 periods
 # Buffers use s24le format, 1000us deadline on core 0 with priority 1
-DAI_ADD(sof/pipe-dai-playback.m4,
+# 2 channels, 48 kHz needed for DAI configuration
+DAI_ADD(sof/pipe-mixer-dai-playback.m4,
 	1, SSP, SSP_NUM, SSP2-Codec,
 	PIPELINE_SOURCE_1, 2, s24le,
-	1000, 1, 0, SCHEDULE_TIME_DOMAIN_DMA)
+	1000, 1, 0, SCHEDULE_TIME_DOMAIN_DMA,
+	2, 48000)
 
-# PCM Media Playback pipeline 3 on PCM 1 using max 2 channels of s32le.
-# 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-pcm-media.m4,
-	3, 1, 2, s32le,
-	1000, 0, 0,
-	8000, 48000, 48000,
-	SCHEDULE_TIME_DOMAIN_DMA,
-	PIPELINE_PLAYBACK_SCHED_COMP_1)
+# FIXME: REMOVE ME
+# # PCM Media Playback pipeline 3 on PCM 1 using max 2 channels of s32le.
+# # 1000us deadline on core 0 with priority 0
+# PIPELINE_PCM_ADD(sof/pipe-low-latency-playback.m4,
+# 	3, 1, 2, s32le,
+# 	1000, 0, 0,
+# 	48000, 48000, 48000,
+# 	SCHEDULE_TIME_DOMAIN_DMA,
+# 	PIPELINE_PLAYBACK_SCHED_COMP_1)
 
 # Connect pipelines together
 SectionGraph."PIPE_NAME" {
 	index "0"
 
 	lines [
 		# media 0
```

Comment:
@plbossart looks good except for the stale comment above. It is no longer the media pipeline